### PR TITLE
Initial Install Doc

### PR DESCRIPTION
**Pull request by adamschaub**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Running Sigma
+
+This guide walks through prerequisites for installing/running Sigma.
+
+## Dependencies
+
+Docker > 17 TODO: What version?
+Go >= 1.7
```
Comment (stephenrlouie) [inline docs/installation.md]: when i view this its on the same line as the docker line

File: docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Running Sigma
+
+This guide walks through prerequisites for installing/running Sigma.
+
+## Dependencies
+
+Docker > 17 TODO: What version?
```
Comment (stephenrlouie) [inline docs/installation.md]: just leave blank if we don't have a specific version?

File: _docs/installation.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+### Run the container
+
+Run Sigma (single instance using the docker tasker)
+
+
+```
+docker run --rm -itd -v /var/lib/sigma:/var/lib/sigma -v /var/run/docker.sock:/var/run/docker.sock sigmadev/sigma [other flags]
+```
+
+`--rm` Remove the sigma container when the process ends
```
Comment (stephenrlouie) [inline _docs/installation.md]: spacing is wrong here. need `-` to force new lines

File: _docs/installation.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+## Pulling/running docker container
+
+### Get the container
+Pull the latest sigma container:
+
+```
+docker pull sigmadev/sigma
+```
+
+Visit hub.docker.com/r/sigmadev/sigma for tagged builds.
```
Comment (meganokeefe) [inline _docs/installation.md]: please link-ify this URL

File: _docs/installation.md (hunk 3 above)
Comment (meganokeefe) [inline _docs/installation.md]: also nit, should be semicolon after `--rm`

Comment (adamschaub) [inline _docs/installation.md]: done

File: _docs/installation.md (hunk 4 above)
Comment (adamschaub) [inline _docs/installation.md]: done

**Pull request merged**